### ISSUE TITLE
Create challenge advertisement material

### ISSUE THREAD

**Issue by tschaffter** [data2health/DREAM-Challenge#14]
**Background:** DREAM Challenges usually have a graphic banner that advertises the challenge on the home page of the wiki. A placeholder for this banner is actually included in the DREAM Challenge Wiki Template which has been used for the [EHR Challenge Staging project](https://www.synapse.org/#!Synapse:syn18405992/wiki/589658). The banner usually includes: 1) the full name of the challenge, 2) one short sentence that describes what the challenge is about, 3) an illustration and 4) the logos of participating organizations.

Task:
1. [x] Provide name of the challenge (tracked here: #10 )
2. [x] Provide list of organizing institutions and partners (tracked here: #12 )
    - [x] Collect logos
3. [x] Provide illustration or ideas of illustrations (ideas tracked in this ticket)
4. [x] Make initial version of the banner (Thomas or X)

**Comment by tschaffter** [comment]
Here are example of banners for past DREAM Challenges:

- https://www.synapse.org/#!Synapse:syn16924919/wiki/583955

**Comment by tschaffter** [comment]
Example of banners from past/current DREAM challenges on Synapse:

- [Malaria DREAM challenge](https://www.synapse.org/#!Synapse:syn16924919/wiki/583955)
- [Tumor Deconvolution DREAM Challenge](https://www.synapse.org/#!Synapse:syn15589870/wiki/582446)
- [IDG-DREAM Drug-Kinase Binding Prediction Challenge](https://www.synapse.org/#!Synapse:syn15667962/wiki/583305)

**Comment by tschaffter** [comment]
@trberg @yy6linda What would be a good illustration to put on the challenge banner? Please feel free to list here any suggestions that you may have.

**Comment by trberg** [comment]
@tschaffter I'm working on a first draft of the banner. I'll post for feedback.

**Comment by tschaffter** [comment]
Awesome! I have experienced working with Adobe Illustrator in case you are using it and have questions.

**Comment by trberg** [comment]
@tschaffter Here is my first attempt at the challenge banner
![DREAM_banner](https://user-images.githubusercontent.com/19157622/54846990-62eec300-4c9a-11e9-88f3-91c66c622aed.png)

**Comment by tschaffter** [comment]
@trberg This banner looks nice! For a production version, the art (cross, chain) could be remade in vector graphics to avoid having pixelized effect. From the bitmap image that you probably have, Adobe Illustrator has a great tool to segment it and create a vector version.

@jguinney I found out that Sage has designers that can do the banner for us if this is a DREAM Challenge. These designers have made the last two example banners that I have included in this ticket. Two comments/remarks:

- Can we brand this challenge as a DREAM Challenge?
- We should come up with the challenge name and scientific questions first before starting to work with the designers.

On a separate note, I have initiated a discussion at Sage to come up with guidelines for the banners that all DREAM projects should abide too. This `standard` should allow us to nicely render a list of challenges if we decide to use the banner for this. We could also come up with standards for a square/circle logo.
